docker/test/integration/resolver/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Helper docker container to run python bottle apps
```

Comment:
Why we need separate docker image instead of just `python:3` which will install this package on startup?